tests/test_bundle.py

```diff
@@ -1009,6 +1012,34 @@ def test_enumeration_bundles(self):
             for x in res.json()['bundles']:
                 self.assertNotIn(x, page_one)
 
+    def test_bundle_enumeration_paging(self):
+        #  Note this test requires a particular /bundle order in s3 test bucket, see test_utils for mocked version
+        with override_bucket_config(BucketConfig.TEST):
+            bucket = Config.get_s3_bucket()
+            key_list = MockStorageHandler.bundle_list
+            for key in key_list:
+                tf = tempfile.TemporaryFile()
+                tf.write(os.urandom(8))
+                s3.upload_fileobj(Fileobj=tf, Bucket=bucket, Key=key)
```

Comment:
The temp file can be avoided, and you can upload a file with 0 size:
```
s3.upload_fileobj(Fileobj=io.BytesIO(b""), Bucket=bucket, Key=key)
```

Reply:
sneaky